### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#295] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110519](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f7f36435-22ff-4ad0-a960-6291fa8cdcff)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/095a5173-f69f-4f6f-8f12-194147c65120)

**Comment by Elia-Belli** [comment]
Un numero $a\in \mathbb{N}$ è privo di inverso aritmetico in $mod\ p^2$ se $(a,p^2)>1$ e l'insieme di tali numeri è: $$\overline{\mathcal{U}(\mathbb{Z}\_{p^2})}=\\{[a]\in \mathbb{Z}\_{p^2}| (a,p^2)>1\\}$$
Ovvero l'insieme di tutti i numeri **non coprimi** con $p^2$, che è esattamente l'insieme complementare di $\mathcal{U}(\mathbb{Z}\_{p^2})$, quindi possiamo ricavare i suoi elementi per differenza insiemistica:
$$\overline{\mathcal{U}(\mathbb{Z}\_{p^2})}=\mathbb{Z}\_{p^2} \setminus \mathcal{U}(\mathbb{Z}\_{p^2})$$
Sappiamo che $|\mathbb{Z}\_{p^2}|=p^2$, mentre $|\mathcal{U}(\mathbb{Z}_{p^2})|=\varphi(p^2)$, allora la cardinalità di $\overline{\mathcal{U}(\mathbb{Z}\_{p^2})}$ è : $p^2-\varphi(p^2)=p^2-(p^2-p^1)=p$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-12 145848](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/5a5c0af3-402e-4125-994e-0ebfd50b3e6b)
